<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackerSettingsDialog</class>
 <widget class="QWidget" name="TrackerSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tracking device settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="textLabel2">
             <property name="minimumSize">
              <size>
               <width>124</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Reference Tool:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="m_referenceToolCombo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="textLabel2_3">
             <property name="minimumSize">
              <size>
               <width>124</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Navigation pointer:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="m_pointerToolCombo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>61</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="m_initializeTrackerButton">
           <property name="text">
            <string>Initialize</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_worldCalibrationMatrixButton">
           <property name="text">
            <string>World calibration matrix...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>61</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListWidget" name="m_toolsListBox">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_addToolButton">
           <property name="text">
            <string>New Tool</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_removeToolButton">
           <property name="text">
            <string>Remove Tool</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_renameToolButton">
           <property name="text">
            <string>Rename Tool</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="spacer3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>16</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCheckBox" name="m_activationCheckBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Active</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QLabel" name="textLabel3">
             <property name="text">
              <string>Description:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacer4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>191</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="m_currentToolDescriptionTextBox">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="m_romFilenameEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_browseButton">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QLabel" name="textLabel1">
             <property name="text">
              <string>Tool Use:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="m_toolUseComboBox">
             <item>
              <property name="text">
               <string>Reference</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Pointer</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ultrasound probe</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Camera</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Generic</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>NoUse</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="m_localCalibrationMatrixButton">
           <property name="text">
            <string>Calibration Matrix</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_calibrateTipButton">
           <property name="text">
            <string>Calibrate Tip</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>m_worldCalibrationMatrixButton</sender>
   <signal>clicked()</signal>
   <receiver>TrackerSettingsDialog</receiver>
   <slot>WorldCalibrationMatrixButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_referenceToolCombo</sender>
   <signal>activated(int)</signal>
   <receiver>TrackerSettingsDialog</receiver>
   <slot>ReferenceToolComboChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_initializeTrackerButton</sender>
   <signal>clicked()</signal>
   <receiver>TrackerSettingsDialog</receiver>
   <slot>InitializeTrackerButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_addToolButton</sender>
   <signal>clicked()</signal>
   <receiver>TrackerSettingsDialog</receiver>
   <slot>AddToolButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_calibrateTipButton</sender>
   <signal>clicked()</signal>
   <receiver>TrackerSettingsDialog</receiver>
   <slot>TipCalibrationButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_removeToolButton</sender>
   <signal>clicked()</signal>
   <receiver>TrackerSettingsDialog</receiver>
   <slot>RemoveToolButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_browseButton</sender>
   <signal>clicked()</signal>
   <receiver>TrackerSettingsDialog</receiver>
   <slot>BrowseRomFileButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_activationCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>TrackerSettingsDialog</receiver>
   <slot>ActiveCheckBoxToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_localCalibrationMatrixButton</sender>
   <signal>clicked()</signal>
   <receiver>TrackerSettingsDialog</receiver>
   <slot>ToolCalibrationMatrixButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_renameToolButton</sender>
   <signal>clicked()</signal>
   <receiver>TrackerSettingsDialog</receiver>
   <slot>RenameToolButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_toolUseComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>TrackerSettingsDialog</receiver>
   <slot>ToolUseComboChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pointerToolCombo</sender>
   <signal>activated(int)</signal>
   <receiver>TrackerSettingsDialog</receiver>
   <slot>NavigationPointerChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_toolsListBox</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>TrackerSettingsDialog</receiver>
   <slot>ToolsListSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>NavigationPointerChanged(int)</slot>
  <slot>ToolsListSelectionChanged()</slot>
 </slots>
</ui>
